parcels/kernels/error.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, particle, fieldset=None, msg=None):
 def parse_particletime(time, fieldset):
     if fieldset is not None and fieldset.U.grid.time_origin != 0:
         # TODO assuming that error was thrown on U field
-        time = fieldset.U.grid.time_origin + timedelta(seconds=time)
+        time = fieldset.U.grid.time_origin + timedelta64(seconds=time)
```

Comment:
Yes, this part wasn't tested in the tests and I forgot it. I've also modified it in the current branch autoAdvanceTime2.
But it has to be changed to `timedelta64(int(time), 's')`. That is the was timedelta64 works.

Reply:
With #319 also fixing this, I'll close #322.